app/models/automated_checks/claim_verifiers/student_loan_plan.rb

```diff
@@ -31,18 +31,24 @@ def student_loans_data
         @student_loans_data ||= StudentLoansData.where(nino:, date_of_birth:)
       end
 
+      def student_loans_data_nino_only
+        @student_loans_data_nino_only ||= StudentLoansData.where(nino:)
```

Comment:
Not sure if it's worth being a bit more explicit here
```suggestion
        @student_loans_data_nino_only ||= StudentLoansData.where(nino:).where.not(date_of_birth:)
```